### PR TITLE
add runtime-id flag

### LINKED ISSUE #11: RuntimeID should be configurable
Currently it's hardcoded:
https://github.com/starfishlabs/oasis-evm-web3-gateway/blob/1f9823ee48879663fa0c8843128d6f7bd0e36cd1/main.go#L29

### PR DESCRIPTION
**Pull request by pro-wh**
fixes #11 

added in roughly the same way the other flags are declared

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -36,6 +34,8 @@ var (
 	nodeAddr string
 	// Ethereum network id
 	chainId uint
```
Comment (pro-wh) [inline main.go:36]: I only see one use each of `nodeAddr` and `chainId`, only to declare the var. they're not used in the actual system, and I don't see other APIs trying to retrieve the values either

Comment (synote) [reply main.go:36]: The chainId is not set into the srvConfig.

File: main.go [hunk 2]
```diff
@@ -36,6 +34,8 @@ var (
...
+	// Runtime ID as used in oasis-net-runner..
```
Comment (kostko) [inline main.go]:
```suggestion
	// Runtime identifier.
```

File: main.go (hunk 1 above)
Comment (ptrus) [reply main.go:36]: Fixed both `nodeAddr` & `chainId` are now used.

**Review by kostko — APPROVED** [review]

**Pull request merged**